LawsonMcCoy/Fading_Light_Narcissus_Lux
# Healthbar

Testing Instructions:

1. Open up TestAI Scene (thank you again, David)
2. Start the game
3. Walk up to Old Man McGee (the cactus) and take damage. This should make the healthbar shake and turn red.
4. NOTE: Currently, damage is -10 per tick. I have partitioned the health into 20s starting from 100, and per -20 the sprite changes and a leaf disappears.
5. Once healthbar works as expected, try sprinting. This should show the stamina bar changing colors as indication of being used.
